scripts/samples/add-file-to-collection.sh

```diff
@@ -15,9 +35,7 @@ echo "$fname"
 EST_HOST="https://shuttle-4.estuary.tech"
 #EST_HOST="https://shuttle-5.estuary.tech"
 
-COLLECTION=""
-
+COLLECTION="185d7ffc-85d8-4b73-86bb-1ca6419bc10d"
 set -x
-#curl --progress-bar -X POST -H "Authorization: Bearer $ESTUARY_TOKEN" -F "data=@$1" -F "name=$fname" $EST_HOST/content/add
-curl --progress-bar -X POST -H "Authorization: Bearer $ESTUARY_TOKEN" -F "data=@$1" -F "name=$fname" $EST_HOST/content/add\?collection="$COLLECTION"
+curl --progress-bar -X POST -H "Authorization: Bearer $ESTUARY_TOKEN" -F "data=@$file" -F "name=$fname" $EST_HOST/content/add\?collection="$COLLECTION"
```

Comment:
the `name` params is now called `filename` - see changes from https://github.com/application-research/estuary/pull/188/files. This affects the `content/add` and `content/add-ipfs` endpoints